sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -16760,7 +16760,7 @@
               "defaultValue": ""
             },
             {
-              "label": "<strong>Will  your partner be at home caring for eligible dependent children on a full-time basis during the your entire study period? </strong>",
+              "label": "<strong>Will your partner be at home caring for eligible dependent children on a full-time basis during your entire study period? </strong>",
```

Comment:
Not part of the PR, just sharing for the benefit of DEVs, the `<strong>` is no longer needed since a global style was introduced that takes care of the same. Adding the `<strong>` or not should not cause any visual difference.